src/Sylius/Behat/Page/Admin/Zone/UpdatePage.php

```
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Zone;
+
+use Behat\Mink\Element\NodeElement;
+use Sylius\Behat\Behaviour\ChecksCodeImmutability;
+use Sylius\Behat\Behaviour\NamesIt;
+use Sylius\Behat\Page\Admin\Crud\UpdatePage as BaseUpdatePage;
+use Sylius\Behat\Page\ElementNotFoundException;
+use Sylius\Component\Addressing\Model\ZoneMemberInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class UpdatePage extends BaseUpdatePage implements UpdatePageInterface
+{
+    use NamesIt, ChecksCodeImmutability;
+
+    /**
+     * @var array
+     */
+    protected $elements = [
+        'code' => '#sylius_zone_code',
+        'name' => '#sylius_zone_name',
+        'type' => '#sylius_zone_type',
+        'member' => '.one.field',
+        'zone_members' => '#sylius_zone_members',
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function countMembers()
+    {
+        try {
+            $selectedZoneMembers = $this->getSelectedZoneMembers();
+
+            return count($selectedZoneMembers);
+        } catch (ElementNotFoundException $exception) {
+            return 0;
```

We should not silence the exception.